database/migrations/2016_06_01_000001_create_oauth_auth_codes_table.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateOauthAuthCodesTable extends Migration
+{
+    /**
+     * The database schema.
+     *
+     * @var \Illuminate\Database\Schema\Builder
+     */
+    protected $schema;
+
+    /**
+     * Create a new migration instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->schema = Schema::connection($this->getConnection());
+    }
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        if (Schema::hasTable('oauth_auth_codes')) return;
+
+        $this->schema->create('oauth_auth_codes', function (Blueprint $table) {
+            $table->string('id', 100)->primary();
+            $table->unsignedBigInteger('user_id')->index();
+            $table->uuid('client_id');
```

Comment:
this shouldn't be `index` ?

Reply:
That's are standard tables from passport. It uses default setting, and i wouldn't like to change that.

Reply:
I think its pointless to put those files in our package 

You can run those migration from packages 

`php artisan migrate --package=vendor/package`

and add this to documentation that you can publish 

[Official documentation](https://laravel.com/docs/master/passport#installation) says you don't need doing anything at all 

![image](https://user-images.githubusercontent.com/214608/111161494-81db7380-859b-11eb-97b9-4790e2173c38.png)


* https://stackoverflow.com/questions/16818240/how-to-automatically-run-the-migrations-in-a-laravel-package
* https://laravel.com/docs/4.2/migrations#running-migrations